src/ergw_aaa_session.erl

```diff
@@ -294,6 +294,12 @@ handle_event(cast, terminate, _State, Data) ->
     terminate_action(Data),
     {stop, normal};
 
+handle_event(info, {'DOWN', _Ref, process, Owner, normal},
+	     _State, #data{owner = Owner} = Data) ->
+    ?LOG(warning, "Received DOWN information for ~p exiting normally", [Data#data.owner]),
```

Comment:
would `info` or `debug` be better, or even remove the log message completely?

Reply:
Depends on why the message is being received. What is sending this? I assume this is a monitored process that has been spawned by the session and is kept under its control, but how does this work exactly? Since it's a normal exit, yes, an info or debug should be ok, but I'd still keep the log just in case.

Reply:
the AAA session process monitors the ergw session process. The original idea was that the AAA process **MUST** terminate before the ergw process.
With the introduction of asynchronous AAA request and the use of those for end-of-session requests it happens that the AAA process is still busy waiting for the response to the stop message when the ergw process terminates.
That is when this log message will trigger.

Reply:
Ah, I see. I think having it may be useful to debug weird timing errors, so I'd settle for notice or info, higher than debug, but yes, less than warning. Info then?

Reply:
yes, `info` is ok